src/createRoom.ts

```diff
@@ -150,6 +150,28 @@ export default async function createRoom(opts: IOpts): Promise<string | null> {
                     [client.getUserId()]: 200,
                 },
             };
+        } else if (opts.roomType === RoomType.UnstableCall) {
+            createOpts.power_level_content_override = {
+                events: {
+                    // Allow all users to send call membership updates
+                    "org.matrix.msc3401.call.member": 0,
+                    // Make calls immutable, even to admins
+                    "org.matrix.msc3401.call": 200,
+                    // Annoyingly, we have to reiterate all the defaults here
+                    [EventType.RoomName]: 50,
```

Comment:
We should define the defaults only once somewhere and import them here. Or does it come from sever side?